green_mbtools/mint/common_utils.py

```diff
@@ -700,6 +701,12 @@ def construct_gdf(args, mycell, kmesh=None):
     Construct Gaussian Density Fitting obejct for a given parameters and unit cell.
     We make sure to disable range-separeting implementation
     '''
+    #pyscf version check for our specific df requirement
+    if(pyscf_version != "2.0.1"):
+      print("###WARNING*****************************************************************###")
+      print("###WARNING POSSIBLE INCOMPATIBILITY OF DF INTEGRALS USE ONLY VERSION 2.0.1 ###")
+      print("###WARNING*****************************************************************###")
+      print("current pyscf version is",pyscf_version," should be 2.0.1")
```

Comment:
Default tab length should be 4 spaces, consistent with elsewhere in the code and repo. The checks are passing because none of the `mint` scripts are covered. Also, not a biggie, but Flake8 standard suggests limiting each line to at most 80 characters for enhanced readability.

Reply:
Changed indentation. Changed lines such that the warning is 80 characters wide (rather than the line, which is longer but still shorter than other lines in this doc).